opentelemetry-jaeger/src/exporter/mod.rs

```diff
@@ -605,8 +605,8 @@ fn build_span_tags(
     }
 
     if status_code != StatusCode::Unset {
-        // Ensure error status is set
-        if status_code == StatusCode::Error {
+        // Ensure error status is set or user manually set it to error
+        if status_code == StatusCode::Error || user_overrides.error {
```

Comment:
I think these should be the opposite: if user_overrides.error is set, then we don't need to set it here as they have already done that. 

This is for the case where the user previously sets one of the fields manually that we are setting for all spans.

Reply:
Spec is a little vogue here(On the one hand the error tag MUST be set to true, on the other hand, it doesn't have to override the previous value). But I agree users should be able to override it
> When Span Status is set to ERROR, an error span tag MUST be added with the Boolean value of true. The added error tag MAY override any previous value.